airgun/browser.py

```diff
@@ -1,12 +1,28 @@
 """Tools to help getting a browser instance to run UI tests."""
 import logging
+import six
+import time
 from copy import copy
 
+from fauxfactory import gen_string
 from selenium import webdriver
 from widgetastic.browser import Browser, DefaultPlugin
 
 from airgun import settings
 
+try:
```

Comment:
why specifically these imports moved out to exception handling sections?

Reply:
As we shouldn't require docker and saucelabs packages for test runs - lib may be used for local runs via local webdriver, docker/saucelabs is rather a case for CI.
I bet you don't have both of them installed (me neither) ;)